neocore/IO/BinaryWriter.py

```diff
@@ -98,8 +98,6 @@ def WriteBytes(self, value, unhex=True):
         if unhex:
             try:
                 value = binascii.unhexlify(value)
-            except TypeError:
```

Comment:
`unhexlify` can only throw this type of exception when using python 2.x. 
Since we require at least 3.4> this code can go and is now handling by `binascii.Error`